blog/_posts/2020-03-18-alumni-interview-jessica.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Alumni Interview with Jessica Greene
+layout: post
+created_at: Weds Mar 18 2020
+permalink: blog/2020-03-18-alumni-interview-jessica
+current: blog
+author: RGSoC Team
+twitter: RailsGirlsSoC
+categories:
+ - alumna-series
+---
+ 
+![](/img/blog/2020/Alumni-Interview-JessicaG.png)
+ 
+> **“My claim to fame is making coffee for the late Christopher Lee!”**
+ 
+Jessica Greene has been living in Berlin for 10 years. After five years in the film & television industry and a stint as a Barista, she took the leap into coding and has since gone on to work as a back-end focused software engineer at Ecosia, the search engine that plants trees.
+ 
+Even if making hot beverages for Saruman himself doesn’t impress you, the following interview detailing Jessica’s time working on the <a href=" https://nextcloud.com/"> Nextcloud </a> project during our last edition of RGSoC in 2018 is bound to.
+ 
+<h4>What first inspired you to get involved with programming? </h4>
+I consider myself a self-taught/community taught developer. After leaving my job at Barn Coffee Roasters in 2017 (and having some much needed time off!) I decided to start my path to becoming a programmer. I had done a very small amount of HTML and worked with flash animations during my art foundation at Leeds art college and my partner works in games who was very supportive and encouraged me to try programming. I started with online courses, mainly web technology and then started to explore a couple of other languages such as Python and GO.
+ 
+<h4>Did you always plan a career in tech? </h4>
+<p>Absolutely not! This is my third professional career and I have had a number of other work experiences. Changing careers is incredibly nerve-racking I was fortunate to have a supportive partner and family as well as being able to gain support from the German government. At first, I thought it would be starting from scratch and that I had nothing to take with me from my past work but that couldn't be further from the truth!</p>
```

Comment:
```suggestion
<p>Absolutely not! This is my third professional career and I have had a number of other work experiences. Changing careers is incredibly nerve-racking. I was fortunate to have a supportive partner and family as well as being able to gain support from the German government. At first, I thought it would be starting from scratch and that I had nothing to take with me from my past work but that couldn't be further from the truth!</p>
```